distribution/lib/Standard/Database/0.0.0-dev/src/Data/SQL.enso

```diff
@@ -158,23 +158,23 @@ optimize_fragments : Vector SQL_Fragment -> Vector SQL_Fragment
 optimize_fragments fragments =
     builder = Vector.new_builder
     go elem =
-        last_part = State.get SQL_Fragment.Code_Part
+        last_part = State.get SQL_Fragment
```

Comment:
`Code_Part` is an _atom constructor_ - that's a singleton. We could allow _atom constructors_ as keys, but then there is problem with argumentless constructors like `Boolean.True` - they are immediately converted to `Atom` during Enso execution and _atoms_ in general aren't singletons.

Restricting keys to _types_ is simpler.